src/components/pages/Home.tsx

```diff
@@ -46,26 +44,43 @@ function SectionQueryResult({
 }) {
   const { config } = useOcean()
   const [result, setResult] = useState<QueryResult>()
+  const [dids, setDIDs] = useState<string>()
 
   useEffect(() => {
     if (!config?.metadataCacheUri) return
-
     const source = axios.CancelToken.source()
+    getHighestLiquidityDIDs().then((results) => {
```

Comment:
you probably tried to find solution for network switching and added refetch for pools here again? Cause they are already fetched and should also refetch down in the `HomePage` component